pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
+| `$dynamicAnchor`            | Core          | 2020-12            | No      | Replaced `$recursiveAnchor`                                                                  |
+| `$dynamicRef`               | Core          | 2020-12            | No      | Replaced `$recursiveRef`                                                                     |
+| `$recursiveAnchor`          | Core          | 2019-09            | 2020-12 | Replaced by `$dynamicAnchor`                                                                 |
+| `$recursiveRef`             | Core          | 2019-09            | 2020-12 | Replaced by `$dynamicRef`                                                                    |
+| `$ref` (legacy)             | Core          | 03                 | 2019-09 | Replaced by `$ref`                                                                           |
+| `$ref`                      | Core          | 2019-09            | No      | Replaced `$ref` (legacy)                                                                     |
+| `$schema`                   | Core          | 03                 | No      | No                                                                                           |
+| `$vocabulary`               | Core          | 2019-09            | No      | No                                                                                           |
+| `additionalItems`           | Core          | 03                 | 2020-12 | Replaced by `items`                                                                          |
+| `allOf`                     | Core          | 04                 | No      | No                                                                                           |
+| `anyOf`                     | Core          | 04                 | No      | No                                                                                           |
+| `const`                     | Validation    | 06                 | No      | No                                                                                           |
+| `contains`                  | Core          | 06                 | No      | No                                                                                           |
+| `contentEncoding` (legacy)  | Core          | 01                 | 04      | Replaced by `media`.`binaryEncoding`                                                         |
+| `media`                     | Hyper-Schema  | 04                 | 07      | Replaced `contentEncoding` (legacy) and replaced by `contentEncoding` and `contentMediaType` |
+| `contentEncoding`           | Validation    | 07                 | No      | Replaced `media`.`binaryEncoding`                                                            |
+| `contentMediaType`          | Validation    | 07                 | No      | Replaced `media`.`type`                                                                      |
```

Comment:
Same here. I don't think we should bother with Hyper-Schema, and this keyword also became an annotation in 2019-09

Reply:
By understanding, the specification covers - Core and Validation. 
The hyperschema is a hypermedia vocabulary—I don't think we should have it here; otherwise, we may have to include other hyperschema keywords not considered for dialect migration.

cc: @jdesrosiers

Reply:
@jdesrosiers I am still waiting to hear from you. Thank you

Reply:
Sorry for the slow response. I've been away almost all of last week and I'm catching up now.

> The hyperschema is a hypermedia vocabulary—I don't think we should have it here

There are three specifications: Core, Validation, and Hyper-Schema. Each spec defines one or more vocabularies.

- Core: core, applicators, unevaluated
- Validation: validation, format-annotation, format-assertion, meta, content
- Hyper-Schema: hyper-schema

So, Hyper-Schema is a specification that defines the hyper-schema vocabulary. I was referring to "Hyper-Schema" the spec.

> otherwise, we may have to include other hyperschema keywords not considered for dialect migration.

I addressed that in a [previous comment](https://github.com/json-schema-org/website/pull/1061#discussion_r1825204529). I suggested including it because it was previously moved from Hyper-Schema to Validation and it didn't make sense to say it replaced the old Hyper-Schema keyword without also listing the keyword it replaced. The comment I linked suggests an alternative.